script/options.js

```diff
@@ -66,9 +66,14 @@ function applyEnvironmentOverrides(options) {
   switch (options.buildtype) {
     case environments.DEVELOPMENT:
     case environments.STAGING:
+      options.move = [{ source: 'vets-robots.txt', target: 'robots.txt' }];
+      options.remove = ['va-robots.txt'];
       break;
 
     case environments.PRODUCTION:
+      options.move = [{ source: 'vets-robots.txt', target: 'robots.txt' }];
+      options.remove = ['va-robots.txt'];
```

Comment:
figured i wouldn't dry this out for now since we're just going to remove it when vets.gov is deprecated.